docs/Users_Guide/overview.rst

```diff
@@ -538,7 +538,7 @@ METplus Components Python Requirements
       programming language
     - 
   * - scikit-image
-    - >=0.16.2
+    - >=0.19.3
```

Comment:
This is listed as 0.18.1 in calcpy requirements.txt. Should we updated calcpy requirements.txt?